RELEASING.md

```diff
@@ -0,0 +1,24 @@
+# Release Steps
+
+To release artifacts 'dependencies-parent', 'dependencies', and 'linkage-checker-enforcer-rules',
+follow the "Developers/Releasing" steps in go/ct4j.
```

Comment:
Please link directly to https://g3doc.corp.google.com/company/teams/cloud-java/tools/developers/releasing.md?cl=head and update that page as needed.

Reply:
Added the link.